documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -6,16 +6,28 @@ Hot Rod is a binary TCP protocol that {brandname} provides for high-performance
 
 .Client intelligence
 
-The Hot Rod protocol includes a mechanism that provides clients with an up to date view of the cache topology.
+The Hot Rod protocol includes a mechanism that provides clients with an up-to-date view of the cache topology.
 Client intelligence improves performance by reducing the number of network hops for read and write operations.
 
+`HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism, and it enables clients to route requests to primary owners, which provides the best performance for Hot Rod clients.
```

Comment:
```suggestion
`HASH_DISTRIBUTION_AWARE` is the default intelligence mechanism and enables clients to route requests to primary owners, which provides the best performance for Hot Rod clients.
```

Reply:
HI @oraNod . Updated. You OK to approve the PR?